aggregate_logging_sizing.adoc

```diff
@@ -0,0 +1,283 @@
+PASS: /run/log/journal/f9370ed252a14f73b014c1301a9b6d1b/system@63c069f54f8d48adada6ef72aad6cd2b-0000000000000001-00053a1b1d414d96.journal
+PASS: /run/log/journal/8cde4877b4ec4bc5962d492606547ac1/system.journal
+----
+
+*Scaling up* +
+
+As you scale up your project, the default logging environment might need some adjustments.
+After upgrading to `systemd-219-22.el7.x86_64`, we've added:
+
+----
+$IMUXSockRateLimitInterval 0
+$IMJournalRatelimitInterval 0
+----
+
+to `/etc/rsyslog.conf` and:
+
+----
+# Disable rate limiting
+RateLimitInterval=1s
+RateLimitBurst=10000
+Storage=volatile
+Compress=no
+MaxRetentionSec=5s
+----
+
+to `/etc/systemd/journald.conf`. 
+
+Now restart the services.
+
+----
+$ systemctl restart systemd-journald.service
+$ systemctl restart rsyslog.service
+----
+
+These settings accommodate for the 'bursty' nature of uploading in bulk. Keep in mind that these tunables are likely to stress the system logging daemons.
+
+Rsyslog is configured (see ratelimit.interval, ratelimit.burst) to rate-limit entries read from the journal at 10,000 messages in 300 seconds. A good rule of thumb is to ensure the rsyslog rate-limits always accommodates the systemd-journald rate-limits.
+
+We can no longer see `systemd-journald[X]: Failed to write entry (Y items, Z bytes), ignoring: Cannot assign requested address` messages. However, this will probably worsen the likelihood of journald file corruption.
+
+
+== Storage Considerations 
+An Elasticsearch (ES) index is a collection of shards and it's correspondent replica shards. This is how ES implements high availability internally, therefore there is little need of using hardware based mirroring RAID variants. RAID 0 can still be used to increase overall disk performance. +
+Every search request needs to hit a copy of every shard in the index, which is fine if every shard is sitting on a different node, but not if many shards have to compete for the same resources.
+Each ES instance requires it's own individual storage, but an OpenShift deployment can only provide volumes shared by all of its pods, which again means that Elastic shouldn't be implemented with a single master. +
+
+A persistent storage volume should be added to each Elasticsearch deployment configuration so that we have one volume per https://www.elastic.co/guide/en/elasticsearch/guide/current/replica-shards.html[replica shard]. On Openshift this is often achieved through https://docs.openshift.com/enterprise/3.2/architecture/additional_concepts/storage.html#persistent-volume-claims[Persistent Volume Claims] +
+
+* 1 volume per shard
+
+* 1 volume per replica shard
+  
+The PVCs must be named based on the `es-pvc-prefix` setting. Refer to https://github.com/openshift/openshift-docs/blob/master/install_config/aggregate_logging.adoc#overview[Persistent Elasticsearch Storage] for more details. Below are some guidelines to help dimensioning your EFK deployment.
```

Comment:
perhaps "Below are capacity planning guidelines for OpenShift Aggregated Logging."